test/scripts/e2e_subs/asset-misc.sh

```diff
@@ -18,7 +18,10 @@ ACCOUNTD=$(${gcmd} account new|awk '{ print $6 }')
 
 ASSET_NAME='Birlot : décollage vs. ࠶🦪'
 
-${gcmd} asset create --creator ${ACCOUNT} --name "${ASSET_NAME}" --unitname amisc --total 1000000000000
+# to ensure IPFS URLs longer than 32 characters are supported
+ASSET_URL="/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/"
+
+${gcmd} asset create --creator ${ACCOUNT} --name "${ASSET_NAME}" --unitname amisc --total 1000000000000 --asseturl "${ASSET_URL}"
```

Comment:
Consider installing "shellcheck" which would complain about the lack of quotes around ${ACCOUNT}.

Our codebase is definitely not totally compliant, but I try to be whenever adding code.

Reply:
Will do!